routes/bucketlist-api-routes.js

```diff
@@ -3,40 +3,61 @@ const db = require("../models");
 module.exports = (app) => {
   //get user's bucktlist
   app.get("/api/bucketlist/:id", (req, res) => {
-    db.Bucketlist.findAll({
-      where: {
-        user_id: req.params.id,
-      },
-    })
-      .then((bucketlist) => {
-        res.json(bucketlist);
+    if (req.user) {
+      db.Bucketlist.findAll({
+        where: {
+          user_id: req.params.id,
+        },
       })
-      .catch((err) => {
-        console.log(err);
-        res.status(500).send("Something went wrong");
-      });
+        .then((bucketlist) => {
+          res.json(bucketlist);
+        })
+        .catch((err) => {
+          console.log(err);
+          res.status(500).send("Something went wrong");
+        });
+    } else {
+      res.status(401).json({ error: "Please log in" });
+    }
   });
   //add one (post)
   app.post("/api/bucketlist", (req, res) => {
```

Comment:
If you want to pass user_id as a params, this api has also it in the url `/api/bucketlist/:id`. You may also need another param to pass in wine_id from a click event on form page

Reply:
understand the value of having the user object in the req. I'll post soon.